components/movementsensor/adxl345/adxl345_test.go

```diff
@@ -97,7 +115,14 @@ func TestInterrupts(t *testing.T) {
 		}
 
 		adxl, err := NewAdxl345(ctx, deps, cfg, logger)
+		test.That(t, err, test.ShouldBeNil)
+
 		interrupt.Tick(context.Background(), true, nowNanosTest())
+		testutils.WaitForAssertion(t, func(tb testing.TB) {
+			readings, err := adxl.Readings(context.Background(), map[string]interface{}{})
+			test.That(tb, err, test.ShouldBeNil)
+			test.That(tb, readings["single_tap_count"], test.ShouldNotBeZeroValue)
+		})
 		test.That(t, err, test.ShouldBeNil)
```

Comment:
This line feels misplaced: `err` hasn't changed since line 118, when we also asserted it should be nil. I'd just remove it.